src/main/java/com/team01/carsharingapp/service/impl/PaymentServiceImpl.java

```diff
@@ -0,0 +1,58 @@
+package com.team01.carsharingapp.service.impl;
+
+import com.team01.carsharingapp.dto.payment.PaymentDto;
+import com.team01.carsharingapp.dto.payment.PaymentRequestDto;
+import com.team01.carsharingapp.mapper.PaymentMapper;
+import com.team01.carsharingapp.model.Car;
+import com.team01.carsharingapp.model.Rental;
+import com.team01.carsharingapp.service.PaymentService;
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@RequiredArgsConstructor
+@Service
+public class PaymentServiceImpl implements PaymentService {
+    private final PaymentMapper paymentMapper;
+    private final Object rentalService;
+
+    @Override
+    public PaymentDto createPayment(PaymentRequestDto requestDto) {
+        //create test entity rental
+        Rental currentRental = new Rental();
+        currentRental.setId(requestDto.rentalId());
+        Car currentCar = new Car();
+        currentCar.setDailyFee(BigDecimal.valueOf(20.0));
+        currentRental.setCar(currentCar);
+        currentRental.setRentalDate(LocalDate.now());
+        currentRental.setReturnDate(LocalDate.now().plusDays(3));
+        //Values for stripe
+        BigDecimal totalPrice = calculateRentalCost(
+                currentRental.getRentalDate(),
+                currentRental.getReturnDate(),
+                currentRental.getCar().getDailyFee());
+        String currentCurrency = requestDto.currency();
+        String currentType = requestDto.type();
```

Comment:
Why do you need it if you are not using it?

Reply:
it's all variables for future impl stripe payment, i write comment in code